cron/workspaceHostTransitions.js

```diff
@@ -106,10 +106,11 @@ async function checkHealth() {
       healthy = false;
     } else {
       try {
-        await request({ uri: url, resolveWithFullResponse: true });
+        const res = await fetch(url);
```

Comment:
Does this conflict with #7205?

Reply:
Yeah, possibly, I'm going to merge that one first and then update here.